sdks/python/apache_beam/typehints/native_type_compatibility_test.py

```diff
@@ -103,6 +103,31 @@ def test_convert_to_beam_types(self):
         native_type_compatibility.convert_to_beam_types(typing_types),
         beam_types)
 
+  def test_is_sub_class(self):
+    self.assertTrue(native_type_compatibility._safe_issubclass(\
```

Comment:
We don't need '\' at the end of the line, since there is a parenthesis. I think you'd need two extra spaces on the next line to avoid lint complaints.
Also, some of these scenarios failed - see PreCommit tests Details page at the end of the PR.
You can also run individual tests via:
```
python ./setup.py test -s apache_beam.typehints.native_type_compatibility_test
```

Reply:
Working on it atm :), thanks for the tips 👍